examples/crypto_ops/encrypt_decrypt.go

```diff
@@ -1,48 +1,47 @@
-// Encrypt/Decrypt example
-// This program demonstrates how to do basic cryptographic operations with the SDKMS Go SDK
 package main
 
 import (
 	"context"
-	"fmt"
 	"log"
-	"net/http"
 
 	"github.com/fortanix/sdkms-client-go/sdkms"
 )
 
-const (
-	myAPIKey string = "N2MwYThlYjgtMGZkNS00OWIxLWFkOWUt..."
-	keyName  string = "AES Key"
-)
+func sample_encrypt_decrypt(client *sdkms.Client, objId string) {
 
-func main() {
-	client := sdkms.Client{
-		HTTPClient: http.DefaultClient,
-		Auth:       sdkms.APIKey(myAPIKey),
-		Endpoint:   "https://sdkms.fortanix.com",
-	}
 	ctx := context.Background()
+	var plainText string = "Hello World!"
+	var key sdkms.SobjectDescriptor
+
+	if objId != "" {
+		key = *sdkms.SobjectByID(objId)
+	} else {
+		key = *sdkms.SobjectByName(keyName)
+	}
+
 	encryptReq := sdkms.EncryptRequest{
-		Plain: []byte("hello, world!"),
+		Plain: []byte(plainText),
 		Alg:   sdkms.AlgorithmAes,
-		Key:   sdkms.SobjectByName(keyName),
+		Key:   &key,
 		Mode:  sdkms.CryptModeSymmetric(sdkms.CipherModeCbc),
 	}
+	log.Printf("Plain Text before encryption %v", plainText)
```

Comment:
I don't know why is part of this PR, but instead of logging the plaintext, assert in the end that it is equal before and after the roundtrip.

Reply:
As far I have seen, we cannot do assert without testing framework in Go.

https://pkg.go.dev/github.com/stretchr/testify/assert#pkg-overview

Reply:
Yes, you can panic if plaintexts differ.